web-admin/src/features/projects/status/ActionsCell.svelte

```diff
@@ -3,59 +3,22 @@
   import * as DropdownMenu from "@rilldata/web-common/components/dropdown-menu";
   import ThreeDot from "@rilldata/web-common/components/icons/ThreeDot.svelte";
   import { ResourceKind } from "@rilldata/web-common/features/entity-management/resource-selectors";
-  import {
-    createRuntimeServiceCreateTrigger,
-    getRuntimeServiceListResourcesQueryKey,
-  } from "@rilldata/web-common/runtime-client";
-  import { runtime } from "@rilldata/web-common/runtime-client/runtime-store";
-  import { useQueryClient } from "@tanstack/svelte-query";
   import { RefreshCcwIcon } from "lucide-svelte";
-  import RefreshResourceConfirmDialog from "./RefreshResourceConfirmDialog.svelte";
 
   export let resourceKind: string;
   export let resourceName: string;
   export let canRefresh: boolean;
-
-  let isConfirmDialogOpen = false;
-  let isDropdownOpen = false;
-  let refreshType: "full" | "incremental" = "full";
-
-  const createTrigger = createRuntimeServiceCreateTrigger();
-  const queryClient = useQueryClient();
-
-  async function refresh() {
-    if (resourceKind === ResourceKind.Model) {
-      await $createTrigger.mutateAsync({
-        instanceId: $runtime.instanceId,
-        data: {
-          models: [
-            {
-              model: resourceName,
-              full: refreshType === "full",
-            },
-          ],
-        },
-      });
-    } else {
-      await $createTrigger.mutateAsync({
-        instanceId: $runtime.instanceId,
-        data: {
-          resources: [{ kind: resourceKind, name: resourceName }],
-        },
-      });
-    }
-
-    await queryClient.invalidateQueries({
-      queryKey: getRuntimeServiceListResourcesQueryKey(
-        $runtime.instanceId,
-        undefined,
-      ),
-    });
-  }
+  export let onOpenRefreshDialog: (
```

Comment:
It feels like this prop name is incorrect given this function is called `on:click` not `on:open`.